pillar/heroku/xpro.sls

```diff
@@ -1,34 +1,54 @@
 {% set app_name = 'xpro' %}
-{% set environments = ['ci', 'rc', 'production'] %}
-{% set status_token = salt.random.get_str(24) %}
+{% set status_token = __vault__:gen_if_missing:secret-operations/global/xpro/django-status-token>data>value %}
```

Comment:
If you're going to set it up here, then it will need to be passed as a string to then get rendered into the YAML, which then gets passed through the Vault renderer. As it is right now it's going to try to execute the `__vault__...` as a function call which will fail.